ui/pages/onboarding-flow/privacy-settings/privacy-settings.js

```diff
@@ -65,18 +65,41 @@ import {
 import IncomingTransactionToggle from '../../../components/app/incoming-trasaction-toggle/incoming-transaction-toggle';
 import { Setting } from './setting';
 
-export default function PrivacySettings() {
-  const t = useI18nContext();
-  const dispatch = useDispatch();
-  const history = useHistory();
-
+/**
+ * Profile Syncing Setting props
+ *
+ * @param {boolean} basicFunctionalityOnboarding
+ * @returns props that are used for the profile syncing toggle.
+ */
+function useProfileSyncingProps(basicFunctionalityOnboarding) {
```

Comment:
This file is big, and I wanted to keep profile syncing stuff in 1 place - hence this function.

Reply:
We could move more stuff for profile sync here, but this does a "good enough" approach for now.